test/data/valid_openapi/custom_headers.json

```diff
@@ -30,13 +30,13 @@
 					"200": {
 						"description": "Ok: Successfully processed the request.",
 						"content": {
-							"application/vnd.retailer.v3+json": {
+							"application/vnd.retailer.v3+xml": {
 								"schema": {
```

Comment:
I've changed the order of content headers in order to check it is giving the preference to JSON first or not.